roles/edpm_pre_adoption_validation/molecule/adoption-service-config-hostname-mismatch/molecule.yml

```diff
@@ -0,0 +1,31 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: podman
+platforms:
+- name: edpm-0
+  command: /sbin/init
+  dockerfile: ../../../../molecule/common/Containerfile.j2
+  image: ${EDPM_ANSIBLE_MOLECULE_IMAGE:-"ubi9/ubi-init"}
```

Comment:
by the way should this be ubi-init?

ubi-init is for when we need to run systmed services in the contaienr.

if we dont actully need that we shoud just use plain ubi

that more  of a nit then anything else.

im just wondering if you are copy pasting this because others used it or do you actully need this

similarly unless you want to mofiy the content of the image you dont need a contienr file and we dont need /sbin/init as the start command but this will also work

Reply:
yeah this was copy paste. I can try to trim it down. But I would like to do that in a separate PR as it is copy pasted to other test cases in this role as well.